distribution/options.html

```diff
@@ -63,6 +63,9 @@
 			<label>
 				<button id="clear-cache">Clear cache</button>
 			</label>
+			<label>
+				<button id="bisect-features">Bisect features</button>
```

Comment:
Let's brand this feature in a way that suggests it can help you find a feature, not just bugs:

```suggestion
				<button id="bisect-features">Find feature</button>
```

Reply:
Likewise, mentions of bugs should be removed. This could be the first message:

```
This process will help you identify what Refined GitHub feature is making changes or causing issues on GitHub. 

x steps remaining, do you still see the change or issue?

Note: You can change page here, but don't use multiple tabs.
```

Reply:
> Let's brand this feature in a way that suggests it can help you find a feature

How about "Pinpoint a feature" for the button text?

Reply:
I think "Find" is more understandable also for international users.